content/contributing-terms.en.md

```diff
@@ -12,7 +12,7 @@ Before adding new terms, open the [`declarations`](./declarations) folder and ch
 
 ### Declaring a new service
 
-Before declaring a service, you will need to choose its name and ID. The ID will be the name of the JSON file in which the service will be declared. It is a normalised version of the service name.
+Before declaring a service, you will need to choose its name and ID. The service ID will be the name of the JSON file in which the service will be declared. It is a normalised version of the service name.
```

Comment:
In order to fix https://github.com/OpenTermsArchive/docs/issues/52, please make sure that every instance of “ID” in the documentation is referred to as “service ID”. For example, in the first sentence of this paragraph 🙂

Reply:
I am thinking of also changing the name at the start to service name